packages/appinio_swiper/lib/types.dart

```diff
@@ -1,8 +1,5 @@
 import 'package:flutter/cupertino.dart';
-import 'enums.dart';
 
-typedef CardsBuilder = Widget Function(BuildContext context, int index);
```

Comment:
This is just Flutter's `IndexedWidgetBuilder`, so I deleted it and used that instead.